executor/src/wamr.rs

```diff
@@ -115,10 +115,10 @@ impl TeaclaveExecutor for WAMicroRuntime {
 
         set_thread_context(Context::new(runtime))?;
 
-        unsafe { wasm_runtime_init() };
+        let ret = unsafe { wasm_runtime_init() };
+        assert!((ret as bool) != false);
```

Comment:
The type of `ret` is `bool`, why do you cast it again?

Reply:
It could be optimized out by rust compiler.
We may consider how to terminate the wasm runtime gracefully in later patch, thanks.

Reply:
My thoughts exactly. I will submit a new PR in the future to address the problem.

Reply:
@qinkunbao if possible, I suggest that the current code could be further slightly improved as follows
```
let ret = unsafe { wasm_runtime_init() as bool };
assert!(ret != false); 
```